anabot/runtime/installation/configuration/root_password.py

```diff
@@ -5,7 +5,8 @@
 from anabot.conditions import is_distro_version
 from anabot.runtime.decorators import handle_action, handle_check, check_action_result
 from anabot.runtime.default import default_handler, action_result
-from anabot.runtime.functions import get_attr, getnode, getparent, getsibling, clear_text
+from anabot.runtime.functions import get_attr, getnode, getparent, getsibling, clear_text, handle_checkbox, \
```

Comment:
`handle_checkbox` and `check_checkbox` are imported, but not used. Is there a reason for not using these existing functions, for example like here? https://github.com/rhinstaller/anabot/blob/main/anabot/runtime/installation/hub/connect_to_redhat/options.py#L156